src/main/java/com/fismobile/appium/pages/AboutPage.java

```diff
@@ -2,42 +2,35 @@
 
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
-import org.openqa.selenium.support.PageFactory;
-import io.appium.java_client.pagefactory.AppiumFieldDecorator;
 import org.openqa.selenium.support.FindBy;
-import com.fismobile.appium.utility.GlobalParameters;
 /*
  *
  * @author wloforte
  */
-public class AboutPage {
+public class AboutPage extends BasePage {
 
     @FindBy(id = "i_am_an_id")
     private WebElement title;
 
     @FindBy(id = "i am a link")
-    private WebElement span;
+    private WebElement link;
 
     @FindBy(id = "comments")
     private WebElement txt_comments;
```

Comment:
Variables in java should follow the camel case style.
Of course the variable names should be more descriptive and unique (please don't define label1, label2, and so on)

Reply:
This variables are just for the sake of the example, we will be more creative and descriptive when naming variables in the actual tests. Will change them anyway.